pkg/vrraw/vrConsts.go

```diff
@@ -0,0 +1,53 @@
+// Package vrraw offers aliases to all VR abbreviations as defined here:
```

Comment:
nit: could you rename this file to just be `vrraw.go`? For small packages, typically I see the convention that the primary go source file contains the same name as the package. Also, I tend to follow the same naming conventions for packages for file names--that is, avoid camelcase (and typically stick to lower case):

https://golang.org/doc/effective_go.html#names

Reply:
No probelm. Will change.